Polygon_mesh_processing/test/Polygon_mesh_processing/test_repair_polygon_soup.cpp

```diff
@@ -289,14 +289,26 @@ void test_simplify_polygons(const bool /*verbose*/ = false)
   points.push_back(Point_3(1,3,0)); // #3
   points.push_back(Point_3(0,1,0)); // #4
   points.push_back(Point_3(1,1,0)); // #5
-  points.push_back(Point_3(0,0,0)); // #6
+  points.push_back(Point_3(0,0,0)); // #6 == #0
 
   // ------
   CGAL_polygon polygon;
-  polygon.push_back(0); polygon.push_back(2); polygon.push_back(4);
+  polygon.push_back(0); polygon.push_back(2); polygon.push_back(4); polygon.push_back(0); polygon.push_back(0);
   polygons.push_back(polygon);
 
   std::size_t res = PMP::internal::simplify_polygons_in_polygon_soup<K>(points, polygons);
+  std::cout << "res: " << res << std::endl;
```

Comment:
@MaelRL Do you want those `std::cout` in CGAL, or should we mark the PR as a draft?

Reply:
This file is just a test file.